### DISCUSSION TITLE
How do I get frigate to save to a different disk?

### DISCUSSION THREAD

**Discussion by TheBigBobo** [community-scripts/ProxmoxVE#1141] [Q&A] [answered]
Hi, I've managed to get frigate installed fine via the helper script, but I want to have it save footage to a different storage device than where it's installed.
I tried following this guide:
https://www.homeautomationguy.io/blog/running-frigate-on-proxmox
But it's a year old and obviously out of date.
![Screenshot from 2024-12-31 19-33-06](https://github.com/user-attachments/assets/9e975fea-2de0-4815-ba90-62242891cea3)
I think I've managed to get the intended location available to the container, but I wasn't able to get frigate to actually use it.
The folder cctv appears in root so I'm guessing something has happened, but trying to edit docker-compose.yml in /opt/frigate by changing ./debug:/media/frigate to /cctv:/media/frigate doesn't appear to do anything. The metrics for storage still show the 20GB I selected when I installed.
I'm very new to all this, so any help would be appreciated.

**Comment by tobz-au** [comment]
hey!  did you manage to sort this by any chance? having the same issues

**Comment by tobz-au** [comment, marked as answer]
OK amazingly Chat GPT helped me.

here is what i did (cctv_clips is my mount point)

Stop the Frigate Recording Manager
`sudo kill -9 540`

Stop the Other Frigate Processes

`sudo kill -9 1444 1445 1448 1450 1452`

Remove existing folder

`sudo rm -rf /media/frigate/recordings`

Create the symlink

`sudo ln -s /cctv_clips /media/frigate/recordings`

Verify the symlink

`ls -l /media/frigate/recordings`

Restart Frigate

Should be working now

Obviously change cctv_clips to whatever your mp=

**Reply by TheBigBobo** [reply]
Perfect. I've never tried ChatGPT, but I'm impressed. Thanks.
